deploy/scripts/src/mobile/test.ts

```diff
@@ -25,27 +25,17 @@ import {
 
 const profiles: ProfileList = {
   smoke: {
-    dcmawPassportIphone: {
+    dcmawStartAuthorizeRequest: {
```

Comment:
This `dcmawStartAuthorizeRequest` profile can just be added, whilst still the `dcmawPassportIphone` and `dcmawDrivingLicenseAndroid` profiles

Reply:
We removed them for the moment. We only need the Authorize Request, and they also have some failures during running (because the FE pages are different now). Will add them as we'll progress with the performance tickets. For the current ticket, this is out of scope. Thanks!

Reply:
You should keep them in and when testing pass in the environment variable flag
```
-e SCENARIOS=dcmawStartAuthorizeRequest
```
which will just run this scenario. See `/deploy/scripts/README.md` for more details on the environment variables.